lib/model/Group.js

```diff
@@ -38,7 +38,7 @@ var Group = new Schema({
     timezone: String,
     commands: Array,
     lazy: Array,
-    active: Array,
+    attributes: Array,
     internalAttributes: Array
 });
```

Comment:
Since I saw you are using the default `_id` property for the stored (`Group`) document identifiers, I think you can remove the `id` property from the `Group` schema, can't you? Just to avoid confusion in the future... :) See: https://github.com/telefonicaid/iotagent-node-lib/blob/task/integrateGroups/lib/model/Group.js#L29